index.html

```diff
@@ -4666,7 +4773,7 @@ <h2 id="AudioBufferSourceNode">
             </p>
             <dl class="parameters">
               <dt>
-                optional double when = 0
+                double when
```

Comment:
Yeah, this is confusing because you only see the diff.  `AudioScheduledSource` has the basic `start(when)` function that is applicable to all subclasses today.  But `AudioBufferSourceNode` has an additional `start(when, offset, duration)` method.  This change just makes `when` required for this 3-arg case. I guess this isn't really necessary, but it does make it slightly clearer which `start` method is being called when you say `start()`.